daprdocs/content/en/reference/cli/cli-overview.md

```diff
@@ -50,7 +50,7 @@ Use "dapr [command] --help" for more information about a command.
 
 ## Command Reference
 
-You can learn more about each Dapr command from the links below.
+You can learn more about each Dapr command from the links below. A full list comparing daprd, CLI, and Kubernetes arguments/annotations can be found [here]({{< ref "arguments-annotations-overview.md" >}}).
```

Comment:
Rethinking this note here. Since you have a note in the specific `run` subcommand and the arguments refer to only that command we can probably remove the note from the overview page not to confuse users.